File: Benchmarking_Languages_Project/Scala_Tests/Test9.scala
var timing = DenseVector.zeros[Long](15)
for(i <- 0 until 15){
	
	val start = System.currentTimeMillis()
	val T9 = DenseMatrix.rand(3000,3000)
	val T9_1 = cholesky(T9)
	val finish = System.currentTimeMillis()
	timing(i) = (finish - start)
}

println(mean(timing)/1000)

File: Benchmarking_Languages_Project/Python_Tests/Test9.py
from scipy import *
from numpy import *
from time import *

timing = zeros(15)
for i in range(15):
	time1 = time()
	T9 = matrix(random.normal(size=(3000,3000)))
	T9 = T9.T * T9
	T9_1 = linalg.cholesky(T9)
	timing[i]=time()-time1
print mean(timing)

